### PR TITLE
[GS-4824] Updating dockerfiles and readme file

### PR DESCRIPTION
**Pull request by ghost**
- Adding jdk20
- Changing components names
- Adding new tags
- Updating readme file

### CODE REVIEW

**Comment by mlouriz** [comment]
Hi @olyagpl can you please review the readme file.

File: README.md [hunk 1]
```diff
@@ -20,33 +31,46 @@ All images support the installation of extra features. The following container i
...
-$ docker pull ghcr.io/graalvm/jdk:java17-22.3.1
+$ docker pull ghcr.io/graalvm/jdk:20.0.1
 ```
 To use GraalVM JDK as base image in Dockerfile, use:
 ```
-FROM ghcr.io/graalvm/jdk:java17-22.3.1
+FROM ghcr.io/graalvm/jdk:20.0.1
 ```
 
 Read more about running GraalVM Community Edition container images [here](https://www.graalvm.org/docs/getting-started/container-images).
```
Comment (olyagpl) [inline README.md:48]:
```markdown
Read more about GraalVM Community Edition container images [here](https://www.graalvm.org/docs/getting-started/container-images/).
```

File: README.md [hunk 2]
```diff
@@ -20,33 +31,46 @@ All images support the installation of extra features. The following container i
...
 
 ## Versioning and Update Policy
 
 To allow users to control their update policy, we provide an update policy based on a tag version.
-To fix the image and allow no updates, you need to use a full version with a build number:
+To fix the image and allow no updates, you need to use a full version with a release date:
+```
+For rpm-based image use:
```
Comment (olyagpl) [inline README.md:55]: For an RPM-based image, use:

File: README.md [hunk 3]
```diff
@@ -20,33 +31,46 @@ All images support the installation of extra features. The following container i
...
+
+ghcr.io/graalvm/$IMAGE_NAME[:][$java_version][-$os_version][-$version][-$date]
 ```
-ghcr.io/graalvm/$IMAGE_NAME[:][$os_version][-$java_version][-$version][-$build_number]
+
+For example, `ghcr.io/graalvm/jdk:20.0.1-ol9-20230613`.
+You can also set an image to a specific java version number that allows an update for a subversion to be pulled.
```
Comment (olyagpl) [inline README.md:61]: a specific Java version

File: README.md [hunk 4]
```diff
@@ -20,33 +31,46 @@ All images support the installation of extra features. The following container i
...
+For instance, using `ghcr.io/graalvm/jdk:latest`, the image will be updated for 22.3.x releases, but not for 22.3.1.   
+Using `ghcr.io/graalvm/native-image` you will always get the latest update available for Native Image, the latest OS which is for now the Oracle Linux 9 and the Oracle Linux 9 slim, the latest Java version.
+
+For Standalone-based image use:
```
Comment (olyagpl) [inline README.md:65]: For a standalone-based image, use:

File: README.md [hunk 5]
```diff
@@ -20,33 +31,46 @@ All images support the installation of extra features. The following container i
...
+
+ghcr.io/graalvm/$IMAGE_NAME[:][$graalvm_version][-$os_version][-$date]
 ```
-For example, `ghcr.io/graalvm/jdk:java17-22.3.1-b1`.
-You can also set an image to a specific version number that allows an update for a subversion to be pulled.
-For instance, using `ghcr.io/graalvm/jdk:java17-22.3`, the image will be updated for 22.3.x releases, but not for 22.3.1.   
-Using `ghcr.io/graalvm/native-image` you will always get the latest update available for Native Image, the latest OS which is for now the Oracle Linux 9 and the Oracle Linux 9 slim, the latest Java version, and the latest GraalVM version.
+
+For example, `ghcr.io/graalvm/graalvm-community:23.0.0-ol9-20230613`.
+You can also set an image to a specific java version number that allows an update for a subversion to be pulled.
```
Comment (olyagpl) [inline README.md:71]: a specific Java version

**Review by ezzarghili — APPROVED** [review]
LGTM

**Pull request merged**